config/initializers/stripe.rb

```diff
@@ -0,0 +1,31 @@
+StripeEvent.configure do |events|
+  events.subscribe 'invoice.payment_succeeded' do |event|
+    StripeEventLog.create(event_id: event.id, event_type, event.type)
+    StripeInvoiceEvents.new(event).payment_succeeded
+  end
+
+  events.subscribe 'invoice.payment_failed' do |event|
+    StripeEventLog.create(event_id: event.id, event_type, event.type)
```

Comment:
Did you mean `event_type: event.type` here? Looks like some tests are failing on a syntax error for this and the other `create`s in this file.

Reply:
Yes, this is a typo. Let me fix it.